--- board.h	2016-02-04 10:39:36.000000000 +0000
+++ board-fixed.h	2016-11-16 10:52:00.000000000 +0000
@@ -372,7 +372,9 @@
 #  include "avr_simulator_uc3/avr_simulator_uc3.h"
 #elif BOARD == USER_BOARD
   // User-reserved area: #include the header file of your board here (if any).
-#  include "user_board.h"
+// changed: ezra buchla 2016-10-29
+// eliminated user_board.h, it was confusing some of the drivers.
+#  include "conf_board.h"
 #elif BOARD == DUMMY_BOARD
 #  include "dummy/dummy_board.h"
 #else
